modules/microshift-olm-deploy-ops-spec-ns.adoc

```diff
@@ -0,0 +1,247 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-OLM-deploy-Operators-specific-namespace_{context}"]
+= Adding OLM-based Operators to a networked cluster in a specific namespace
+
+Use this procedure if you want to specify a namespace for an Operator, for example, `olm-microshift`. In this example, the catalog is scoped and available in the global `openshift-marketplace` namespace. The Operator uses content from the global namespace, but runs only in the `olm-microshift` namespace. For {microshift-short} clusters that have network connectivity, Operator Lifecycle Manager (OLM) can access sources hosted on remote registries.
+
+[IMPORTANT]
+====
+All of the Operators installed in a specific namespace must have the same watch scope. In this case, the watch scope is *All namespaces*.
```

Comment:
@ShaunaDiaz `All namespaces` should be `own namespace`

Reply:
Ah, so the Operator can still use content from the `openshift-marketplace` namespace even if the watch scope is `own namespace`?